aspnetcore/signalr/get-started-signalr-core.md

```diff
@@ -0,0 +1,107 @@
+---
+uid: signalr/get-started-signalr-core
+title: Get Started with SignalR on ASP.NET Core
+author: rachelappel
+ms.author: rachelap
+description: In this tutorial, you will create a SignalR on ASP.NET Core application.
+manager: wpickett
+ms.date: 02/23/2018
+ms.topic: tutorial
+ms.technology: dotnet-signalr
+ms.prod: aspnet-core
+ms.custom: mvc
+---
+# Tutorial: Get Started with SignalR for ASP.NET Core
+
+By [Rachel Appel](https://twitter.com/rachelappel)
+
+This tutorial teaches the basics of building a real-time app using SignalR for ASP.NET Core. 
+
+This tutorial demonstrates the following SignalR development tasks:
+
+> [!div class="checklist"]
+> * Add the SignalR library to an ASP.NET web application.
+> * Create a SignalR hub to push content to clients.
+> * Modify the `Startup` class to configure the application.
+> * Use the SignalR JavaScript library to send messages and display updates from the hub.
+
+# Prerequisites
+
+Install the following:
+
+* [.NET Core 2.0.0 SDK](https://www.microsoft.com/net/core) or later.
+* [Visual Studio 2017](https://www.visualstudio.com/downloads/) version 15.6 or later with the ASP.NET and web development workload.
+
+## Create an ASP.NET Core project that uses SignalR
+
+1. Use the **File** > **New Project** menu option and choose **ASP.NET Core Web Application**.
+
+  ![New Project dialog in Visual Studio](get-started-signalr-core/_static/signalr-new-project-dialog.png)
+
+1. In the next step, choose **Web Application** to use a lightweight project template for using Razor Pages. Then select **Ok**.
+
+  ![New Project dialog in Visual Studio](get-started-signalr-core/_static/signalr-new-project-choose-type.png)
+
+1. Add the `Microsoft.AspNetCore.SignalR` NuGet package to the ASP.NET Core project. You can do this by selecting **Tools** > **NuGet Package Manager** > **Package Manager Console**. In the console, issue  the following command:
+
+  ```text
+  Install-Package Microsoft.AspNetCore.SignalR -Version 1.0.0-alpha2-final
+  ```
+
+  This command installs the `Microsoft.AspNetCore.SignalR` NuGet package that contains libraries SignalR server-side code.
+
+1. Install the JavaScript client library by using npm. `npm install` creates a *\node_modules\* folder and sub-folders for the libraries in the same location as where you run the npm command.
+
+  ```
```

Comment:
- Add "console" after the 3 back ticks.
- I recommend specifying "--save" in the install command, so that the dependency is saved in *package.json*.